app/index.js

```diff
@@ -41,11 +41,13 @@ module.exports = yeoman.generators.Base.extend({
       chalk.red('Maxxton Microservice generator\n\n') +
       yosay() + '\n\n' +
       '==============================\n' +
-      'Version: 1.0\n' +
+      'Version: 1.1\n' +
```

Comment:
Maybe update the package.json so it also holds the right version (currently at version 0.0.0.)